src/makeCalculator.js

```diff
@@ -4,7 +4,27 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calc = {
+    result: 0,
+    add: (n) => (calc.result += n),
+    subtract: (n) => (calc.result -= n),
+    multiply: (n) => (calc.result *= n),
+    divide: (n) => (calc.result /= n),
+
+    operate: function (callback, number) {
+      callback(number);
+
+      return this;
```

Comment:
[CODE KNOWLEDGE] - when defining methods within an object, you should use the shorthand method syntax instead of the `function` keyword.

BAD EXAMPLE:
 operate: function (callback, number) {
  // your code
 },

GOOD EXAMPLE:
 operate(callback, number) {
  // your code
 },